### DISCUSSION TITLE
sonoff nspanel pro non si aggiorna

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#3122] [GENERALE - Discussioni generiche]
ho appena comprato NSpanel pro per usarlo con HA e di base era presente la versione 1.4 del firmware .. aggiorno alla versione 1.5 ed ora non mi propone le versioni successive.
come se fosse ultima versione la 1.5.0
come posso fare per aggiornarlo..
grazie

**Comment by KC-inDomus** [comment]
normale: è spiegato in guida che attivare ADB impedisce per sempre l'aggiornamento firmware a versioni superiori a quella presente al momento dell'attivazione di ADB. motivo per cui o non lo si fa, o se lo si fa prima si aggiorna all'ultimissima versione disponibile.
c'è però [un salvagente](https://indomus.it/guide/aggiornare-via-ota-il-firmware-di-un-sonoff-nspanel-pro-con-sideloading-attivo-adb-sbloccato/).

**Reply by effe-7** [reply]
Ma io ancora non ho attivato ADB

**Reply by KC-inDomus** [reply]
ah. allora non so che dirti, pensavo avessi già sbloccato. prova a resettarlo

**Comment by effe-7** [comment]
problema aggiornamento risolto praticamente senza far niente ... stamani era possibile farlo!
forse un problema lato sonoff!! comunque sia risultato ottenuto...
grazie
